lib/assets/junit.xml.erb

```diff
@@ -6,7 +6,7 @@
 
 <testsuites tests="<%= number_of_tests %>" failures="<%= number_of_failures %>">
   <% @results.each do |testsuite| %>
-    <testsuite name=<%= testsuite[:target_name].encode(:xml => :attr) %> tests="<%= testsuite[:number_of_tests] %>" failures="<%= testsuite[:number_of_failures] %>" time="<%= testsuite[:duration] %>">
+    <testsuite name=<%= (testsuite[:target_name] == nil ? testsuite[:test_name] : testsuite[:target_name]).encode(:xml => :attr) %> tests="<%= testsuite[:number_of_tests] %>" failures="<%= testsuite[:number_of_failures] %>" time="<%= testsuite[:duration] %>">
```

Comment:
You can use `testsuite[:target_name].nil?` instead 👍  The rest looks great, thanks for your contribution